sdks/python/apache_beam/io/jdbc.py

```diff
@@ -120,8 +120,8 @@ def default_io_expansion_service(classpath=None):
         ('password', str),
         ('connection_properties', typing.Optional[str]),
         ('connection_init_sqls', typing.Optional[typing.List[str]]),
-        ('write_statement', typing.Optional[str]),
         ('read_query', typing.Optional[str]),
+        ('write_statement', typing.Optional[str]),
```

Comment:
Is this failing a lint test somewhere ?

Reply:
I don't think the order really matters here, but I might be missing something.

Reply:
I tried using the query param of ReadFromJdbc, which I found out was ignored all the time. Then I tried to debug what was received on java side as configuration and found that query is set as 'writeStatement' rather than 'readQuery'. I saw that the order in which it's defined in python is different than that's [defined in java](https://github.com/apache/beam/blob/master/sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcSchemaIOProvider.java#L67). I tried to change the order and then the query parameter started working.